CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## main / unreleased
 
 * [FEATURE] Templates: Add urlQueryEscape to template functions. #17403
+* [ENHANCEMENT] TSDB: Add Option to configure TSDB Block Reload Interval using `block_reload_interval` #16728
```

Comment:
Please remove this, as it overlaps with the release notes section in the PR description.